src/components/organisms/interactiveTable/InteractiveTable.tsx

```diff
@@ -126,6 +136,14 @@ export const InteractiveTable = <ColumnKey extends string>({ columns, defaultSor
     );
   };
 
+  const renderCellDisplay = (cell: Record<ColumnKey, TInteractiveTableCell>[ColumnKey]) => {
```

Comment:
Suggestion: I have a bit of an aversion to render methods contained within a component because it gets redefined on every render. It looks like only the `showValues` value is being used from within the context of this component so maybe we could pass that in when we call the render method if we put it outside the `InteractiveTable` parent?

Reply:
Good shout 🙂  Fixed